packages/workflow/package.json

```diff
@@ -0,0 +1,58 @@
+{
+	"name": "@wordpress/workflow",
+	"version": "0.1.0",
+	"description": "Workflow palette for running abilities in WordPress.",
```

Comment:
I think workflows and command palette are two different things. I don't mind this "experimental package" that combines but let's make it "private: true" to avoid this package leaking. 

I also think the command palette shouldn't be listing "abilities", it should list "workflows" that use abilities.